text.go

```diff
@@ -325,12 +325,12 @@ func (t *TextDisplay) SetTextColor(color Color) {
 	C.go_fltk_TextDisplay_set_text_color((*C.Fl_Text_Display)(t.ptr()), C.uint(color))
 }
 
-// TextColor gets the default color of text in the widget.
+// TextFont gets the default color of text in the widget.
 func (t *TextDisplay) TextFont() Font {
 	return Font(C.go_fltk_TextDisplay_text_font((*C.Fl_Text_Display)(t.ptr())))
 }
 
-// SetTextColor sets the default color of text in the widget.
+// SetTextFont sets the default color of text in the widget.
```

Comment:
You've fixed the name of the method called, but the description is still wrong... :(

Reply:
Sorry, I've fixed them now!